packages/amplify-graphql-api-construct/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@aws-amplify/graphql-construct-alpha",
-  "version": "0.9.0",
+  "name": "@aws-amplify/graphql-api-construct",
+  "version": "1.0.0",
```

Comment:
If we update this manually, lerna is going to bump it to `1.1.0` or `1.0.1` depending on the changes.